acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ThreadBesuNodeRunner.java

```diff
@@ -134,26 +133,21 @@ public void startNode(final BesuNode node) {
             .withMetricsSystem(metricsSystem)
             .build();
 
-    final BesuController<?> besuController;
-    try {
-      besuController =
-          builder
-              .synchronizerConfiguration(new SynchronizerConfiguration.Builder().build())
-              .dataDirectory(node.homeDirectory())
-              .miningParameters(node.getMiningParameters())
-              .privacyParameters(node.getPrivacyParameters())
-              .nodePrivateKeyFile(KeyPairUtil.getDefaultKeyFile(node.homeDirectory()))
-              .metricsSystem(metricsSystem)
-              .transactionPoolConfiguration(TransactionPoolConfiguration.builder().build())
-              .ethProtocolConfiguration(EthProtocolConfiguration.defaultConfig())
-              .clock(Clock.systemUTC())
-              .isRevertReasonEnabled(node.isRevertReasonEnabled())
-              .storageProvider(storageProvider)
-              .targetGasLimit(GasLimitCalculator.DEFAULT)
-              .build();
-    } catch (final IOException e) {
-      throw new RuntimeException("Error building BesuController", e);
-    }
+    final BesuController<?> besuController =
+        builder
+            .synchronizerConfiguration(new SynchronizerConfiguration.Builder().build())
+            .dataDirectory(node.homeDirectory())
+            .miningParameters(node.getMiningParameters())
+            .privacyParameters(node.getPrivacyParameters())
+            .nodePrivateKeyFile(KeyPairUtil.getDefaultKeyFile(node.homeDirectory()))
```

Comment:
Is there any change here except removing try/catch block?

Reply:
not sure why this is no longer required - will look into it - but yes, its the removal of the try/catch